spin.toml

```diff
@@ -215,3 +170,13 @@ environment = { DESTINATION = "https://developer.fermyon.com/spin/kv-store-tutor
 [component.trigger]
 route = "/spin/kv-store"
 executor = { type = "wagi" }
+
+
+# Redirecting from common to spin incase someone had it bookmarked before removal of common
+[[component]]
+id = "redirect-contributing-common-to-spin"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/cli-reference" }
+[component.trigger]
+route = "/common/cli-reference"
+executor = { type = "wagi" }
```

Comment:
@itowlson Do you think we need to keep this around or is this something we can remove?

Reply:
Leave it in case there are any dangling links to it